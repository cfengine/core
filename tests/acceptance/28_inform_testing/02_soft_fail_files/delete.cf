body file control
{
  inputs => { "$(sys.policy_entry_dirname)/../common_soft_fail.cf.sub" };
}

bundle common testcase
{
  vars:
    "filename" string => "$(this.promise_filename)";
}

bundle agent setup
{
  files:
    "$(G.testfile)"
      create => "true";
}

bundle agent main
{
  files:
    "$(G.testfile)"
    delete => init_delete;
}
