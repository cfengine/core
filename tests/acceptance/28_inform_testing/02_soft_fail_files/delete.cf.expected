    info: files promise '/tmp/TEST.cfengine' repaired, delete => init_delete
    info: Deleted file '/tmp/TEST.cfengine'
